quad_simulator/a1_description/xacro/gazebo.xacro

```diff
@@ -0,0 +1,324 @@
+<?xml version="1.0"?>
+<robot>
+    <!-- ros_control plugin -->
+    <gazebo>
+        <plugin name="gazebo_ros_control" filename="libgazebo_ros_control.so">
+            <robotNamespace>/a1</robotNamespace>
+            <robotSimType>gazebo_ros_control/DefaultRobotHWSim</robotSimType>
+        </plugin>
+        <plugin filename="libground_truth_estimator.so" name="ground_truth_estimator">
+            <updateRateHZ>500.0</updateRateHZ>
+        </plugin>
+    </gazebo>
+    <!-- Show the trajectory of trunk center. -->
+    <!-- <gazebo>
```

Comment:
Remove the commented section?

Reply:
We might want to leave it there first, that's used by the original a1 description from unitree

Reply:
We can make decision to leave it or not during refactoring.